packages/services/src/__tests__/boost-award.service.test.ts

```diff
@@ -75,6 +75,393 @@ describe("BoostAwardService", () => {
     mockReset(mockUserService);
   });
 
+  describe("awardForStake", () => {
```

Comment:
I'm surprised to see new tests showing up in this PR?  I guess that's fine though